src/Ast/PhpProperty.php

```diff
@@ -18,13 +18,16 @@
 
 namespace Google\Generator\Ast;
 
+use Google\Generator\Utils\ResolvedType;
+
 /** A property within a class. */
 final class PhpProperty extends PhpClassMember
 {
     public function __construct(string $name)
     {
         $this->name = $name;
         $this->value = null;
+        $this->type = null;
```

Comment:
As of PHP 8.2, [setting dynamic properties is deprecated](https://php.watch/versions/8.2/dynamic-properties-deprecated). This repo is currently on PHP 8.1, which is why no errors are thrown. We can wait until we update the repo to PHP 8.2 to add these properties, or we could add it now, e.g. 

```php
final class PhpProperty extends PhpClassMember
{
    private ?ResolvedType $type;
```

... but since we'll have to update these across the whole library eventually, I'll leave that up to you whether you'd like to define it now or not.

Reply:
Ehhh let's change it once and for all.

Reply:
If it's between introducing a "consistent" change that goes against a best practice, and introducing an "inconsistent" change that follows best practices, I'd prefer to go with the latter

Reply:
...and I think having undefined properties is a really bad practice

Reply:
Woops, I have this change but did not included the file on the commit. Will fix it rn.

Reply:
Woops, I have this change but didn't include the file on the Commit. Fixing it now.